server/src/test/java/org/elasticsearch/discovery/SnapshotDisruptionIT.java

```diff
@@ -59,9 +57,10 @@ public void testDisruptionOnSnapshotInitialization() throws Exception {
             .put(DiscoverySettings.COMMIT_TIMEOUT_SETTING.getKey(), "30s") // wait till cluster state is committed
             .build();
         final String idxName = "test";
-        configureCluster(settings, 4, 2);
+        configureCluster(settings, 4);
         final List<String> allMasterEligibleNodes = internalCluster().startMasterOnlyNodes(3);
         final String dataNode = internalCluster().startDataOnlyNode();
+        setMinimumMasterNodes(2);
```

Comment:
I think this shouldn't be necessary because the cluster knows it only has three master nodes so will already have set `minimum_master_nodes` to 2.